docs/source/var-handlers.rst

```diff
@@ -0,0 +1,155 @@
++------------------+----------------+-----------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| levels           | dictionary     | {name: plev19, units: Pa, e3sm_axis_name: plev}     | Distinguishes model-level variables, which require remapping from the default model  level to the level defined in the levels dictionary.                                                                                                                                                                                                                                       |
++------------------+----------------+-----------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Where are variable handlers defined?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The supported CMIP6 variables and their handlers are defined in the `E3SM CMIP6 Data Conversion Tables Confluence page`_.
+
+.. _E3SM CMIP6 Data Conversion Tables Confluence page: https://acme-climate.atlassian.net/wiki/spaces/DOC/pages/858882132/CMIP6+data+conversion+tables
+
+These handler definitions are transferred to the ``e3sm_to_cmip`` repository at the following locations:
+
+1. ``e3sm_to_cmip/cmor_handlers/handlers.yaml``
+
+    - **Stores handler definitions for atmosphere and land variables.**
+    - Each top-level entry defines a handler.
+    - There can be more than one handler per variable (e.g., to handle different frequencies)
+
+    - Example:
+
+    .. code-block:: yaml
+
+        -   name: pr
+            units: kg m-2 s-1
+            raw_variables: [PRECT]
+            table: CMIP6_day.json
+            unit_conversion: null
+            formula: PRECT * 1000.0
+            positive: null
+            levels: null
+        -   name: pr
+            units: kg m-2 s-1
+            raw_variables: [PRECC, PRECL]
+            table: CMIP6_Amon.json
+            unit_conversion: null
+            formula: (PRECC + PRECL) * 1000.0
+            positive: null
+            levels: null
+
+2. ``e3sm_to_cmip/cmor_handlers/vars`` directory
+    - Each handler is defined as Python module (``.py`` file). This is the legacy design for  defining handlers, which is progressively being be replaced by ``handlers.yaml``.
```

Comment:
Maybe change to `which is progressively being replaced by`

Reply:
Fixed